backend/routers/plugins.py

```
@@ -109,3 +119,104 @@ def get_plugin_money_made(plugin_id: str):
+    data = json.loads(plugin_data)
+    data['approved'] = False
+    data['id'] = data['name'].replace(' ', '-').lower()
+    data['uid'] = uid
+    if data['private']:
+        data['id'] = data['id'] + '-private'
+        if private_plugin_id_exists_db(data['id'], uid):
+            data['id'] = data['id'] + '-' + ''.join([str(random.randint(0, 9)) for _ in range(5)])
+    else:
+        if public_plugin_id_exists_db(data['id']):
+            data['id'] = data['id'] + '-' + ''.join([str(random.randint(0, 9)) for _ in range(5)])
+    os.makedirs(f'_temp/plugins', exist_ok=True)
```

Ruff

f-string without any placeholders

Remove extraneous `f` prefix

(F541)